package.json

```diff
@@ -38,8 +38,6 @@
     "bump:playground:version": "cd playground && npm --no-git-tag-version --allow-same-version version $VERSION",
```

Comment:
you can even remove this and also https://github.com/asyncapi/asyncapi-react/blob/master/playground/package.json#L3 as we do not really need to version playground, as it is just dev env, no plans to push to npm

Reply:
Sure. I removed them.